<ui version="4.0" >
 <class>MultiLoadBase</class>
 <widget class="QDialog" name="MultiLoadBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Multi-module load</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <widget class="Q3GroupBox" name="listBox" >
       <property name="title" >
        <string>List of modules to be loaded</string>
       </property>
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="Q3Table" name="moduleTable" >
            <property name="numRows" >
             <number>0</number>
            </property>
            <property name="numCols" >
             <number>3</number>
            </property>
            <column>
             <property name="text" >
              <string>Modules/files to load                        </string>
             </property>
            </column>
            <column>
             <property name="text" >
              <string>stave/sector SN</string>
             </property>
            </column>
            <column>
             <property name="text" >
              <string>stave/sector pos.</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QPushButton" name="addButton" >
              <property name="text" >
               <string>Add Module</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer3" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>16</width>
                <height>21</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="addRootButton" >
              <property name="text" >
               <string>Add ROOT file</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QPushButton" name="addListButton" >
              <property name="text" >
               <string>Add List of Modules</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer3_2" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>16</width>
                <height>21</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="addStaveButton" >
              <property name="text" >
               <string>Add Stave/Sector</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QPushButton" name="removeButton" >
              <property name="text" >
               <string>Remove</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer4" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>26</width>
                <height>16</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="upButton" >
              <property name="enabled" >
               <bool>true</bool>
              </property>
              <property name="text" >
               <string>Move up</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer5" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>26</width>
                <height>21</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="downButton" >
              <property name="enabled" >
               <bool>true</bool>
              </property>
              <property name="text" >
               <string>Move down</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="Q3GroupBox" name="dataBox" >
       <property name="title" >
        <string>Data matters (does not affect ROOT files)</string>
       </property>
       <layout class="QHBoxLayout" >
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QLabel" name="textLabel1" >
            <property name="text" >
             <string>Location of module folders</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLineEdit" name="pathToModules" />
            </item>
            <item>
             <widget class="QPushButton" name="browseButton" >
              <property name="text" >
               <string>Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QComboBox" name="meastypeBox" >
              <property name="minimumSize" >
               <size>
                <width>180</width>
                <height>0</height>
               </size>
              </property>
              <item>
               <property name="text" >
                <string>user defined</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="measIndex" >
              <property name="minimum" >
               <number>1</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="restrStdTest" >
              <property name="text" >
               <string>Reduced List</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QCheckBox" name="getAmbush" >
              <property name="text" >
               <string>upload ambush from:</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="ambushDir" >
              <property name="enabled" >
               <bool>true</bool>
              </property>
              <property name="text" >
               <string>data/ambush/</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QCheckBox" name="savetoRoot" >
              <property name="enabled" >
               <bool>true</bool>
              </property>
              <property name="text" >
               <string>save data to a ROOT file into</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="pathToRoot" >
              <property name="enabled" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="Q3GroupBox" name="anaBox" >
       <property name="title" >
        <string>Analysis</string>
       </property>
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QCheckBox" name="runPDB" >
            <property name="text" >
             <string>run PDB analysis</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="textLabel1_2" >
              <property name="text" >
               <string>restricted set of cuts:</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cutsBox" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="uploadBox" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>Upload results to PDB</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="loadNeededFiles" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>Load only data that is analysed</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="removeAfterPDB" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>Remove data after PDB analysis</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QCheckBox" name="multiPlots" >
              <property name="text" >
               <string>produce bi-stave summary plots</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <item>
               <spacer name="spacer7" >
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType" >
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>16</width>
                  <height>16</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QCheckBox" name="savePlots" >
                <property name="enabled" >
                 <bool>false</bool>
                </property>
                <property name="text" >
                 <string>save plots to</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="plotsPath" >
                <property name="enabled" >
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="plotsBrwsButton" >
                <property name="enabled" >
                 <bool>false</bool>
                </property>
                <property name="text" >
                 <string>Browse</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QPushButton" name="okButton" >
         <property name="text" >
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer1" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>30</width>
           <height>21</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton" >
         <property name="text" >
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Table</class>
   <extends>Q3Frame</extends>
   <header>q3table.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>getAmbush</sender>
   <signal>toggled(bool)</signal>
   <receiver>ambushDir</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>runPDB</sender>
   <signal>toggled(bool)</signal>
   <receiver>cutsBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>runPDB</sender>
   <signal>toggled(bool)</signal>
   <receiver>multiPlots</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>savetoRoot</sender>
   <signal>toggled(bool)</signal>
   <receiver>pathToRoot</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>multiPlots</sender>
   <signal>toggled(bool)</signal>
   <receiver>plotsBrwsButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>multiPlots</sender>
   <signal>toggled(bool)</signal>
   <receiver>plotsPath</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>runPDB</sender>
   <signal>toggled(bool)</signal>
   <receiver>removeAfterPDB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>multiPlots</sender>
   <signal>toggled(bool)</signal>
   <receiver>runPDB</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>multiPlots</sender>
   <signal>toggled(bool)</signal>
   <receiver>savePlots</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>runPDB</sender>
   <signal>toggled(bool)</signal>
   <receiver>uploadBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>runPDB</sender>
   <signal>toggled(bool)</signal>
   <receiver>loadNeededFiles</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
